res/css/views/rooms/_EventTile.scss

```diff
@@ -89,20 +89,28 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
         margin-left: 64px;
     }
 
-    &.mx_EventTile_info {
-        padding-top: 1px;
-    }
-
     .mx_EventTile_avatar {
         top: 14px;
         left: 8px;
         cursor: pointer;
         user-select: none;
     }
 
-    &.mx_EventTile_info .mx_EventTile_avatar {
-        top: $font-6px;
-        left: $left-gutter;
+    &.mx_EventTile_info {
+        padding-top: 0;
+
+        .mx_EventTile_avatar,
+        .mx_EventTile_e2eIcon {
+            top: 2.25px; // align with mx_EventTile_line
```

Comment:
We really ought to avoid fractional px values, can we not use align-items/align-self/some other vertical alignment tool?

Reply:
Yes definitely. The reason why I chose fractional values is perhaps that all of them required a little bit complicated way. Let me try again.